static/etc/apt/auth.conf.d/01-fips.conf.example

```diff
@@ -0,0 +1,3 @@
+# replace the <LOGIN> and <PASSWORD> with your own private access
```

Comment:
Do comments work on netrc syntax? Or maybe that only works for apt. Which is a nice thing. No warnings?

Reply:
Actually good question, I just assumed here - I didn't have the comments while testing

Reply:
I added a line stating the comments should be removed